lianad/src/database/sqlite/mod.rs

```diff
@@ -481,6 +481,23 @@ impl SqliteConn {
         .expect("Database must be available")
     }
 
+    pub fn insert_external_coins<'a>(
```

Comment:
Lets keep the function name payjoin specific.
Something like: `inserts_outpoint_seen_before`